api/PositionOptions.json

```diff
@@ -116,13 +63,61 @@
           "deprecated": false
         }
       },
+      "secure_context_required": {
+        "__compat": {
+          "description": "Secure context required",
+          "support": {
+            "chrome": {
+              "version_added": "47"
+            },
+            "chrome_android": {
+              "version_added": "47"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "55"
+            },
+            "firefox_android": {
+              "version_added": "55"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
       "enableHighAccuracy": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/PositionOptions/enableHighAccuracy",
           "support": {
-            "webview_android": {
```

Comment:
This change (sorting the browsers?) seems to be unrelated to the topic of the PR. I don't mind incidental clean ups in the course of another change, but for future reference, it's easier to understand and review when those changes happen in their own commits